cli/running/running.go

```diff
@@ -317,3 +319,15 @@ func Logrotate(cmdCtx *cmdcontext.CmdCtx) (string, error) {
 	// Rotates logs [instance name pid]
 	return fmt.Sprintf("Logs has been rotated. PID: %v.", pid), nil
 }
+
+// Check returns the result of checking the syntax of the application file.
+func Check(appFilePath string, cmdCtx *cmdcontext.CmdCtx) (stdout []byte, stderr []byte, err error) {
```

Comment:
Overcomplicated.
The `appFilePath` is specified in `cmdCtx`, no need to pass it twice.
It's enough to return an error. If it `== nil`, everything is fine, otherwise all necessary information can be passed in the error.

Reply:
* Fixed
* Made `func Check(cmdCtx *cmdcontext.CmdCtx) error `.  In order to pass a message with a description of the syntax error from Lua to Go, I had to create a custom `error` with `checkError struct`